requirements.txt

```diff
@@ -8,6 +8,8 @@ matplotlib >= 2.2, < 3.0; python_version >= '2.7' and python_version < '3.0'
 matplotlib >= 2.2, < 3.1; python_version >= '3.5' and sys_platform == 'win32'
 matplotlib >= 2.2; python_version >= '3.5' and sys_platform != 'win32'
 networkx
+llvmlite >= 0.31, < 0.32; python_version < '3.6'
+numba < 0.48; python_version < '3.6'
 numba
```

Comment:
shouldn't there be a python version constraint on the second line as well?
afaik, none of these get processed in order...

Reply:
They don't? I got lucky then! I'll fix that.

Reply:
Hm but that makes the `llvmlite` situation tricky. `numba==0.47.0` has `llvmlite>=0.31` in it's `requirements.txt` which means pip will install `0.32` if it does numba first.

Reply:
except for `llvmlite` they are listed alphabetically. would be weird if that worked out by chance until now :)

Reply:
True! I just couldn't believe there's no way to specify install order! (without something silly like `cat requirements.txt | xargs pip install -`